.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
why not run prettier directly?

Reply:
Running prettier directly was causing the formatted code to be commited, but after the commit the unformatted code came back after the commit